docs/constraints.md

```diff
@@ -1,6 +1,7 @@
 # Constraints
 
 As well as matchers opsdroid also has constraints. There are decorators for your functions which prevent the skill from being called even if it is matched by a matcher.
+There are also inverted constraints. They allow to call all skills except the ones matched by matcher.
```

Comment:
This isn't clear I'm afraid. Perhaps something like this would be better.

```suggestion
You can also invert constraints so that they filter out the opposite of the criteria.
```
```

Reply:
Thank you for the suggestion!